src/main/resources/freemarker/new-progress-report-case.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Broad Data Use Oversight System - New Progress Report ready for your vote</title>
+</head>
+
+<body style="text-align: center; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100%; height: 100%; margin: 0; padding: 0;">
+<div style="text-align: left; margin: 0 auto; width: 600px;">
+    <table style="width: 600px">
+        <tr>
+            <td style="padding: 15px 15px 0px; text-align: center;">
+                <img src="https://duos.org/images/favicon/android-chrome-192x192.png" height="100px" width="100px" alt="DUOS Logo">
+            </td>
+        </tr>
+        <tr>
+            <td id="userName" style="padding: 15px 15px 0px; font-family: 'Montserrat', sans-serif; font-size: 22px; color: #1F3B50; text-align: left; line-height: 25px; display: block; font-weight: 500;">
+                Hello ${userName},
+            </td>
+        </tr>
+        <tr>
+            <td style="padding: 15px 15px 0px; font-family: 'Montserrat', sans-serif; font-size: 16px; color: #1F3B50; text-align: justify; line-height: 25px;">
+                ${electionType} Review case id ${entityName}, has been created. Please click on the following link to log your vote on it:
```

Comment:
I realize that this is essentially a clone of the DAR version. I don't think `${electionType}` needs to be injected here, I think it will always be a Data Access type since that's all that we have currently. There is the RP election type, but we don't send separate emails for that type, nor should we.